Workbooks/CosmosDb/Mongo/Highest RU consumption.workbook

```diff
@@ -0,0 +1,1254 @@
+{
+  "version": "Notebook/1.0",
+  "items": [
+    {
+      "type": 1,
+      "content": {
+        "json": "\r\n<span style=\"font-size:40px;\"> Why is my Highest RU Consuming Shard spiking to 100% ? </span>\r\n\r\n<br>\r\n\r\n## Step 1. Let's understand what Highest RU Consuming Shard means in Cosmos DB"
```

Comment:
I'd suggest changing the title to "Optimize my API for MongoDB cost and performance"

Reply:
Since customers will likely come from a higher level mindset rather than questioning a single graph.